### PR TITLE
Development merge request; updates for EAC-CPF 2.0

### PR DESCRIPTION
**Pull request by fordmadox**
Final updates for EAC-CPF 2.0 release.

### CODE REVIEW

**Comment by fordmadox** [comment]
Converted this pull request to a draft since we still need to update a few of the markdown files.

**Review by karinbredenberg — APPROVED** [review]
The files and folders that is named source can be misunderstood for having to do with the element named source....
The intro.xml wonder about having names there? I would suggest having a contribution page in our wiki instead of one in each repository. 
Do we need to have the vendor files in our repository or can we point to them instead?

**Comment by fordmadox** [comment]
@karinbredenberg :  I've added a new commit that addresses the first two issues.  for now, we still need to have the vendor directory, to make local testing easier for everyone, but we could look into other options for that in the future.  anyhow, I think this should be good to go now.  take a look next week and let me know if it's fine to merge.  once the schematron part is done, I'll then prepare this release.

File: src/release-info/intro.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  
+  EDITORS: Society of American Archivists Technical Subcommittee on Encoded Archival Standards (TS-EAS)
+  
+  PUBLISHER: Society of American Archivists in collaboration with the Staatsbibliothek zu Berlin
+  
+  PURPOSE: The EAC-CPF Schema is a communication standard for the description of corporate bodies,
+  persons, and families. It is based on the International Council of Archives' International Standard 
+  Archival Authority Record For Corporate Bodies, Persons and Families (ISAAR(CPF)). For additional
+  documentation on the use of the standard, see the EAC-CPF Tag Library
+  (http://eac.staatsbibliothek-berlin.de/schemata-and-tag-library/).
```
Comment (karinbredenberg) [inline src/release-info/intro.xml]: https

File: src/release-info/intro.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+   
+  *****************************************************************
+ 
+  This schema has been generated automatically. 
+   
+  If you would like to contribute to this project, please see:
+  https://github.com/SAA-SDT/eac-cpf-schema/blob/development/CONTRIBUTING.md
```
Comment (karinbredenberg) [inline src/release-info/intro.xml]: Should be: https://github.com/SAA-SDT/TS-EAS-subteam-notes/wiki/Contributing-to-the-EAS-standards

File: src/release-info/intro.xml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  
+  Comments, questions, and suggestions may be addressed to: 
+  ts-eas@archivists.org
+  
+  For more information about the EAC-CPF team, see https://github.com/SAA-SDT/TS-EAS-subteam-notes/wiki/EAC-CPF-Sub-Team
```
Comment (karinbredenberg) [inline src/release-info/intro.xml]: Should be: https://github.com/SAA-SDT/TS-EAS-subteam-notes/wiki/EAC-CPF-Sub-Team

File: src/release-info/intro.xml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                              An XML Schema:
+  Encoded Archival Context - Corporate Bodies, Persons, and Families (EAC-CPF)
+
+  See: https://eac.staatsbibliothek-berlin.de/schemata-and-tag-library/
+
+  *****************************************************************
+  *                                                               *                
+  *                            EAC-CPF 2.0                        *
+  *                                                               *
+  *                           2022 July 17                        *
```
Comment (karinbredenberg) [inline src/release-info/intro.xml]: Not sure about the date, should it be 3rd of August or the day council approved (May 18th)?

**Review by karinbredenberg — APPROVED** [review]
Some comments on the intro have been made.

File: src/release-info/intro.xml (hunk 4 above)
Comment (fordmadox) [reply src/release-info/intro.xml]: I'm leaning toward just leaving it out altogether.  Let's discuss tomorrow.

File: src/release-info/intro.xml (hunk 2 above)
Comment (fordmadox) [reply src/release-info/intro.xml]: Got it, thanks.

File: src/release-info/intro.xml (hunk 3 above)
Comment (fordmadox) [reply src/release-info/intro.xml]: That's what was there previously, right?

Comment (fordmadox) [reply src/release-info/intro.xml]: In any event, I've added the link to a new line for now.

File: src/release-info/intro.xml (hunk 1 above)
Comment (fordmadox) [reply src/release-info/intro.xml]: Updated.  Thanks!

**Review by karinbredenberg — APPROVED** [review]
All changes needed are fixed.

**Pull request merged**